### DISCUSSION TITLE
Migration docs/tool

### DISCUSSION THREAD

**Discussion by johannperez** [dotnet/maui#144] [General]
## Summary

It would be nice to have docs as thorough as possible on how to migrate Xamarin.Forms project into MAUI. May be even a tool to transform your csproj/references/namespaces?

As far as I can see there are so many points still open which may (will?) make an automated migration tool not possible, still, for the sake of the community/companies betting on Xamarin right now easing the transition seems imperative.

## Intended Use Case

Long running/big projects already in prod built on Xamarin.Forms

----

PS: More than glad to help on this one given the chance

**Comment by lazmeister** [comment]
Would be a great tool!
